src/model/user/Admin.java

```diff
@@ -0,0 +1,17 @@
+package model.user;
+
+public class Admin extends User{
+    public Admin(String firstName, String lastName, String username, String email, String password) {
```

Comment:
[Fix] aceste doua clase, Admin si Customer nu aduc nimic in plus fata de User. Daca singura diferenta dintre ele e doar in nume, se putea adauga un enum cu UserType si sa fie ADMIN si CUSTOMER si adaugat in clasa User. Extindem o clasa pentru a adauga functionalitate extra. Un Admin poate avea un camp permisssions care e List<Permission> si acela e un enum care contine READ, WRITE, MODIFY, DELETE, si diferiti admini pot avea diferite permisiuni. Sau Customer ar putea avea List<Reservation>